pkg/migrate/migrate_test.go

```diff
@@ -3068,3 +3095,766 @@ func Test_copyAllPVCs(t *testing.T) {
 		})
 	}
 }
+
+func Test_validateVolumeAccessModes(t *testing.T) {
+	for _, tt := range []struct {
+		name            string
+		srcStorageClass string
+		dstStorageClass string
+		deletePVTimeout time.Duration
+		podReadyTimeout time.Duration
+		wantErr         bool
+		resources       []runtime.Object
+		input           map[string]corev1.PersistentVolume
+		expected        map[string]map[string]PVCError
+	}{
+		{
+			name: "should pass due to no accessmode errors",
+			input: map[string]corev1.PersistentVolume{
+				"pv0": {
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "pv0",
+					},
+					Spec: corev1.PersistentVolumeSpec{
+						StorageClassName: "srcSc",
+						ClaimRef: &corev1.ObjectReference{
+							Name: "pvc",
+						},
+					},
+				},
+			},
+			expected:        make(map[string]map[string]PVCError),
+			srcStorageClass: "srcSc",
+			dstStorageClass: "dstSc",
+			resources: []runtime.Object{
+				&storagev1.StorageClass{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "srcSc",
```

Comment:
Could we try to add names here that clarifies WHEN.. THEN .. EXPECTED? 
more info: https://dev.to/smyrman/test-with-expect-a-bdd-style-go-naming-pattern-5eh5

Reply:
Good suggestion. I'll give the TWE naming a convention a try.